tests/test_hugo_server_readiness.py

```
@@ -0,0 +1,161 @@
+    """进程自然退出时 _monitor_logs 记录退出日志。"""
+    mock_process = MagicMock()
+    # readline returns "" immediately (process stdout closed = process exited)
+    mock_process.stdout.readline.return_value = ""
+    mock_process.poll.return_value = 1
+    manager.process = mock_process
+    manager.is_running = True
+    manager.stop_log_thread = False
+
+    manager._monitor_logs()
+
+    exit_logs = [l for l in manager.logs if "已退出" in l["message"]]
```

Ruff (0.15.18)

[error] Ambiguous variable name: `l`

(E741)